webpack.config.js

```diff
@@ -0,0 +1,11 @@
+const { merge } = require("webpack-merge");
+
+const commonConfig = require("./webpack/webpack.common.config");
+
+module.exports = (env) => {
+  const browser = env.chrome ? "chrome" : "firefox";
```

Comment:
Could this be done in an array to check if the browser selected in the env file is supported?
As this might cause env files with different browsers to qualify as firefox

Reply:
Firefox <3
You're right. Added a validation check for `env` to allow only currently supported browsers.

**Clarification**: There is no env file management in this PR. I use webpack env variables in the relative npm scripts to achieve it.